charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: spire-server
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: spire-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spire-server
+  serviceName: spire-server
+  template:
+    metadata:
+      namespace: {{ .Values.namespace }}
+      labels:
+        app: spire-server
+    spec:
+      serviceAccountName: spire-server
+      shareProcessNamespace: true
+      containers:
+        - name: spire-server
+          # image: gcr.io/spiffe-io/spire-server:0.11.0
+          # image: tsidentity/tornjak-spire-server:latest
+          image: tsidentity/tornjak-spire-server:{{ .Values.spireVersion }}
```

Comment:
since we do this for the registrar maybe let's create image variables for tornjak image and the agent image?